changelogs/unreleased/gh_10155_make_iproto_resistant_to_misusage.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/core
+
+* Fixed a bug when a server could crash if a client sent an IPROTO replication
+  request without waiting for pending requests to complete (gh-10155).
```

Comment:
Nit: the commit message should reference the original commit from master branch (the `cherry-picked from ... line`). Besides, you mention the commit from your own branch instead of the one merged into master.

To make it right you should do:
`git cherry-pick -x 45f42c79f2be17d98ef557bdbfacf6426dbbdf29`, https://github.com/tarantool/tarantool/commit/45f42c79f2be17d98ef557bdbfacf6426dbbdf29 is the right hash

The resulting commit message will be
```

    iproto: make iproto resistant to misusage
    
    Fixed an issue when the tarantool could be easily crashed using iproto
    incorrectly, not according to the protocol.
    
    Closes #10155
    
    NO_DOC=bugfix
    
    (cherry picked from commit 45f42c79f2be17d98ef557bdbfacf6426dbbdf29)

```

Same about the other PR #10893

Reply:
Fixed.

Reply:
Thanks!